keadm/app/cmd/util/cloudinstaller.go

```diff
@@ -178,26 +206,15 @@ func linesFromReader(r io.Reader) ([]string, error) {
 //RunEdgeController starts edgecontroller process
 func (cu *KubeCloudInstTool) RunEdgeController() error {
 
-	filetoCopy := fmt.Sprintf("cp %s/kubeedge/cloud/%s /usr/local/bin/.", KubeEdgePath, KubeCloudBinaryName)
-	cmd := &Command{Cmd: exec.Command("sh", "-c", filetoCopy)}
-	err := cmd.ExecuteCmdShowOutput()
-	errout := cmd.GetStdErr()
-	if err != nil || errout != "" {
-		fmt.Println("in error")
-		return fmt.Errorf("%s", errout)
-
-	}
-	binExec := fmt.Sprintf("chmod +x /usr/local/bin/%s && %s > %s/kubeedge/cloud/%s.log 2>&1 &", KubeCloudBinaryName, KubeCloudBinaryName, KubeEdgePath, KubeCloudBinaryName)
-	cmd = &Command{Cmd: exec.Command("sh", "-c", binExec)}
+	binExec := fmt.Sprintf("chmod +x %skubeedge/cloud/%s && %skubeedge/cloud/%s > %skubeedge/cloud/%s.log 2>&1 &", KubeEdgePath, KubeCloudBinaryName, KubeEdgePath, KubeCloudBinaryName, KubeEdgePath, KubeCloudBinaryName)
+	cmd := &Command{Cmd: exec.Command("sh", "-c", binExec)}
 	cmd.Cmd.Env = os.Environ()
 	env := fmt.Sprintf("GOARCHAIUS_CONFIG_PATH=%skubeedge/cloud", KubeEdgePath)
 	cmd.Cmd.Env = append(cmd.Cmd.Env, env)
-	err = cmd.ExecuteCmdShowOutput()
-	errout = cmd.GetStdErr()
-	if err != nil || errout != "" {
-		return fmt.Errorf("%s", errout)
+	err := cmd.StartCmd()
+	if err != nil {
+		fmt.Println("in error")
```

Comment:
can we log the error ? also can we shift to using log package instead of fmt for all logging ?